pilot/pkg/config/kube/crd/types.go

```diff
@@ -199,7 +199,7 @@ var knownTypes = map[string]schemaType{
 		schema: model.ServiceRole,
 		object: &ServiceRole{
 			TypeMeta: meta_v1.TypeMeta{
-				Kind:       model.ServiceRole.Type,
+				Kind:       "ServiceRole",
```

Comment:
This file is auto-generated, why is it edited?

Reply:
I think someone do it manually previously. This is output when I rerun the script. I can remove this from PR as my api change doesn't affect it.